pkg/apis/toolchain/v1alpha1/tektoninstallation_types.go

```diff
@@ -21,6 +22,14 @@ type TektonInstallationStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	// Conditions is an array of current TektonInstallation conditions
+	// Supported condition types:
+	// TektonInstalled, FailedToInstallTekton
```

Comment:
Need to correct type name -> Installed, FailedToInstall

Reply:
Even type is one `TektonReady` and Installed, FailedToInstall are reason

Reply:
addressed in c3c3805